### PR TITLE
Bc player

### PR DESCRIPTION
**Pull request by RevinKey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
+    "@brightcove/react-player-loader": "^1.2.0",
+    "cross-fetch": "^3.0.2",
+    "global": "^4.3.2",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-redux": "^7.0.3",
     "react-scripts": "3.0.0",
-    "redux": "^4.0.1"
+    "redux": "^4.0.1",
+    "redux-thunk": "^2.3.0",
```
Comment (ldayananda) [inline package.json]: fyi, I got an error because of the comma on this line when trying to run

File: src/actions.js [hunk 2]
```diff
@@ -10,69 +10,70 @@ export const RECEIVE_VIDEO_ANALYTICS = 'RECEIVE_VIDEO_ANALYTICS';
 /** Action Creators */
 
 export function requestVideoList(accountId = '6027103981001') {
-  return {
-    type: REQUEST_VIDEO_LIST,
-    accountId
-  };
+    return {
```
Comment (ldayananda) [inline src/actions.js:13]: I think you might have editor settings that have 4 spaces instead of 2

File: src/components/BrightcovePlayer.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import {connect} from "react-redux";
+import Player from '@brightcove/react-player-loader';
+
+class BrightcovePlayer extends Component {
+
+    success = ({ ref }) => {
+        this.playerRef = ref;
+        if (this.props.selectedVideo !== null) {
+            this.playerRef.catalog.load({
+                sources: [this.props.selectedVideo]
```
Comment (ldayananda) [inline src/components/BrightcovePlayer.js:12]: `selectedVideo` is a videoId right? Is this the sanctioned way of doing this?

File: src/components/BrightcovePlayer.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        this.playerRef.catalog.getVideo(this.props.selectedVideo, (error, video) => {
+            this.playerRef.catalog.load(video);
+        });
+    }
+    render() {
+        return (
+            <Player attrs={{id: 'videoPlayer'}}
+                    accountId='6027103981001'
+                    playerId='default'
+                    onSuccess={this.success}
```
Comment (ldayananda) [inline src/components/BrightcovePlayer.js:29]: it seems like we could include `videoId: selectedVideo` here and not have to call the catalog method above nor need the code in `componentDidUpdate`?

Comment (misteroneill) [reply src/components/BrightcovePlayer.js:29]: That seems true, though, not sure how that would affect video switching. Probably won't want to re-render the component when switching videos.

File: src/components/BrightcovePlayer.js (hunk 3 above)
Comment (misteroneill) [reply src/components/BrightcovePlayer.js:12]: Sure, this seems fine.

File: src/components/BrightcovePlayer.js (hunk 4 above)
Comment (ldayananda) [reply src/components/BrightcovePlayer.js:29]: 🤔 yeah I guess the re-rendering might be wasteful

**Review by ldayananda — APPROVED** [review]

**Pull request merged**